modules/apps/depot/depot-service/src/main/java/com/liferay/depot/internal/security/permission/wrapper/DepotPermissionCheckerWrapper.java

```diff
@@ -311,6 +318,8 @@ private boolean _isGroupOwner(Group group) throws PortalException {
 			return false;
 		}
 
+		group = StagingUtil.getLiveGroup(group);
```

Comment:
We usually use :
_Group liveGroup = StagingUtil.getLiveGroup(group);_

to prevent possible side effects.

please change it on the 3 places.
thanks!